src/routines/routines.ycp

```diff
@@ -640,7 +641,10 @@ define list<map> ReadHardware(string hwtype) {
     /* Confirmation: label text (detecting hardware: xxx) */
     string hwstring = _("All Network Devices");
     if(haskey(hwstrings, hwtype)) hwstring = hwstrings[hwtype]:"";
-    if(!Confirm::Detection(hwstring, "yast-lan")) return [];
+
+    // don't ask in the 1st stage, if we need to save a netcard config then network was already
+    // used for installation and asking is useless
```

Comment:
when you start DVD installation normally then detection is done at the beginning of installation and the popup is not raised.

Try to start installer with manual=1 (this is case for s390) and will see it as well